### ISSUE TITLE
[Feature] Assign default scrapers to individual fields and add support to scrape all in one go

### ISSUE THREAD

**Issue by VonTittyslappen** [stashapp/stash#715]
**Is your feature request related to a problem? Please describe.**
Some scrapers carry inaccurate data or data which breaks country flag for example

**Describe the solution you'd like**
I was thinking (dangerous I know), but it would be great if you were able to set default scrapers for individual fields, so for example, you could have IAFD scrape the Date Of Birth, and use a different scraper to source other information, and allow support where you just hit a 'Scrape All' button and it scrapes the performer details from all the assigned default scrapers in one go. I hope I've explained that properly, let me know if not. This would save so much time where people are having to run one scraper at a time and selecting the fields that they do want to import from and the ones they dont. Being able to assign default scrapers for certain fields would be a massive help when it comes to performers

**Comment by WithoutPants** [comment]
Possible alternative proposal:

Add a "aggregate" type of scraper. This scraper would map fields to other scrapers. When the scraping operation is performed, it runs all of the related scrapers and populates the fields accordingly. We would need a way to hide scrapers from the UI as well, so that the user could add scrapers that are not used directly - only via aggregate scrapers.

**Comment by VonTittyslappen** [comment]
yeah, aggregate scrapers works too :) its a nice feature to have eventually
